pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -39,10 +39,11 @@ type ApplicationBackupSpec struct {
 	ReclaimPolicy      ApplicationBackupReclaimPolicyType `json:"reclaimPolicy"`
 	SkipServiceUpdate  bool                               `json:"skipServiceUpdate"`
 	// Options to be passed in to the driver
-	Options          map[string]string `json:"options"`
-	IncludeResources []ObjectInfo      `json:"includeResources"`
-	ResourceTypes    []string          `json:"resourceTypes"`
-	BackupType       string            `json:"backupType"`
+	Options             map[string]string `json:"options"`
+	IncludeResources    []ObjectInfo      `json:"includeResources"`
+	ResourceTypes       []string          `json:"resourceTypes"`
+	BackupType          string            `json:"backupType"`
+	CSISnapshotClassMap map[string]string `json:"csiSnapshotClassMap"`
```

Comment:
Can you please add comment on the purpose of the field?

Reply:
Done